echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/bearychat/BearychatService.groovy

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.echo.bearychat
+
+import retrofit.http.Body
+import retrofit.http.GET
+import retrofit.http.POST
+import retrofit.client.Response
+import retrofit.http.Query
+
+interface BearychatService {
+
+  @GET("/v1/user.list")
```

Comment:
Any reason to use `.` in place of `/`? I think the latter is preferred.

Reply:
I found that bearychat put their API on the github. And i just use them without further thinking.  You can see their open api [here](https://github.com/bearyinnovative/OpenAPI/tree/master/api).